lib/templates/template-1.yml

```
@@ -3,16 +3,62 @@ name: Votacions AFFaC
 organization: 
   default_locale: ca
   users_registration_mode: enabled
-participatory_spaces:
+  available_locales: 
+    - ca
+    - es
+  force_users_to_authenticate_before_access_organization: false
+scopes:
+  - code: type
+    name:
+      ca: Tipus de consulta
+      es: Tipo de consulta
+content_blocks:
+  - manifest_name: hero
+    weight: 1
+    scope_name: homepage
+    images_container:
+      - name: background_image
+        file: template-1/hero_background_image.png
+        content_type: image/png
+    settings:
+      welcome_text:
+        ca: Hola!
+        es: Hola!
```

```suggestion
        ca: %{organization_name}
        es: %{organization_name}
```
probably something like that

---

Done.